dotenv/godotenv_test.go

```diff
@@ -368,16 +375,19 @@ func TestParsing(t *testing.T) {
 	parseAndCompare(t, "export\tOPTION_A=2", "OPTION_A", "2")
 	parseAndCompare(t, "  export OPTION_A=2", "OPTION_A", "2")
 	parseAndCompare(t, "\texport OPTION_A=2", "OPTION_A", "2")
+	parseAndCompare(t, `export OPTION_A="export A"`, "OPTION_A", "export A")
 
 	// it 'expands newlines in quoted strings' do
 	// expect(env('FOO="bar\nbaz"')).to eql('FOO' => "bar\nbaz")
 	parseAndCompare(t, `FOO="bar\nbaz"`, "FOO", "bar\nbaz")
+	parseAndCompare(t, `FOO=a\tb`, "FOO", `a\tb`)
+	parseAndCompare(t, `FOO="a\tb"`, "FOO", "a\tb")
```

Comment:
Should we test `\\[bvfr]` escapes too?

Reply:
Added a mega escape case in f3671f66ef9965a6350c4ce2227d31cae07352b1.